scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -348,7 +356,7 @@ task ExtractTask {
         --filter-set-name ~{filter_set_name}'
     fi
 
-    gatk --java-options "-Xmx9g" \
+    gatk --java-options "-Xmx~{memory_gib - 3}g" \
```

Comment:
3g is a lot of overhead for the VM. Probably could get away with -1 here?

Reply:
The baseline code was allotting at least 3GiB, or whatever the override value - 9... ☹️